common/src/main/java/me/confuser/banmanager/common/commands/CommonCommand.java

```diff
@@ -155,31 +156,40 @@ public CommandParser getParser(String[] args) throws NoSuchMethodException, Ille
 
   public List<String> handlePlayerNameTabComplete(CommonSender sender, String[] args) {
     ArrayList<String> mostLike = new ArrayList<>();
+    if(args.length == 1) {
+      if (isValidNameDelimiter(args[0])) {
+        String[] names = splitNameDelimiter(args[0]);
 
-    if (isValidNameDelimiter(args[0])) {
-      String[] names = splitNameDelimiter(args[0]);
+        String lookup = names[names.length - 1];
 
-      String lookup = names[names.length - 1];
-
-      if (plugin.getConfig().isOfflineAutoComplete()) {
-        for (CharSequence charSequence : plugin.getPlayerStorage().getAutoCompleteTree().getKeysStartingWith(lookup)) {
-          mostLike.add(args[0] + charSequence.toString().substring(lookup.length()));
+        if (plugin.getConfig().isOfflineAutoComplete()) {
+          for (CharSequence charSequence : plugin.getPlayerStorage().getAutoCompleteTree().getKeysStartingWith(lookup)) {
+            mostLike.add(args[0] + charSequence.toString().substring(lookup.length()));
+          }
         }
-      }
-
-    } else if (plugin.getConfig().isOfflineAutoComplete()) {
-      for (CharSequence charSequence : plugin.getPlayerStorage().getAutoCompleteTree().getKeysStartingWith(args[0])) {
-        mostLike.add(charSequence.toString());
-      }
-    } else {
-      CommonPlayer senderPlayer = sender instanceof CommonPlayer ? (CommonPlayer) sender : null;
-      String lower = args[0].toLowerCase();
-      for (CommonPlayer player : plugin.getServer().getOnlinePlayers()) {
-        if ((senderPlayer == null || senderPlayer.canSee(player)) && player.getName().toLowerCase().startsWith(lower)) {
-          mostLike.add(player.getName());
+      } else if (plugin.getConfig().isOfflineAutoComplete()) {
+        for (CharSequence charSequence : plugin.getPlayerStorage().getAutoCompleteTree().getKeysStartingWith(args[0])) {
+          mostLike.add(charSequence.toString());
+        }
+      } else {
+        CommonPlayer senderPlayer = sender instanceof CommonPlayer ? (CommonPlayer) sender : null;
+        String lower = args[0].toLowerCase();
+        for (CommonPlayer player : plugin.getServer().getOnlinePlayers()) {
+          if ((senderPlayer == null || senderPlayer.canSee(player)) && player.getName().toLowerCase().startsWith(lower)) {
+            mostLike.add(player.getName());
+          }
         }
       }
     }
+    if(args.length == 2) {
```

Comment:
Should this not be > 1? It's possible to use multiple reasons, i.e. `/ban confuser #test #hello`